src/components/FilterBadges/index.tsx

```diff
@@ -11,15 +14,18 @@ import { SeeMoreModal } from "./SeeMoreModal";
 const { convertObjectToArray } = array;
 const { parseQueryString } = queryString;
 
+const hiddenQueryParams = [
```

Comment:
Great use case for a [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set)!